src/engine/bufferscalers/enginebufferscalest.cpp

```diff
@@ -17,27 +17,30 @@ using namespace soundtouch;
 namespace {
 
 // Due to filtering and oversampling, SoundTouch is some samples behind.
-// The value below was experimental identified using a saw signal and SoundTouch 1.8
-// at a speed of 1.0
-// 0.918 (upscaling 44.1 kHz to 48 kHz) will produce an additional offset of 3 Frames
-// 0.459 (upscaling 44.1 kHz to 96 kHz) will produce an additional offset of 18 Frames
-// (Rubberband does not suffer this issue)
-const SINT kSeekOffsetFrames = 519;
+// The value below was experimental identified using a saw signal and SoundTouch 2.1.1
+constexpr SINT kSeekOffsetFramesV20101 = 429;
+
+// From V 2.3.0 Soundtouch has no initial offset at unity, but it is to early with
+// lowered pitch and to early with raised pitch ~+-2000 farmes = 50 ms.
```

Comment:
I have added the description of my test. 
Does it help?

Reply:
I appreciate the comment, but it didn't really help. I have a couple questions:
1. what exactly do you mean with offset in this comment? Latency expressed as a number of frames/samples?
2. how can that offset be earlier (lowered/raised pitch) than no offset (at unity (which I assume means no change in pitch)).

Reply:
1 In my test with extrem pitches, the saw tooth was shifted by up to 2000 frames compared to the unity signal. This is relative to the pitch, but I have not investigated how exactly. 

2 It can be earlier, because of the settling samples. Soundtouch needs some samples to be settled before you can consume the output even at unity. Mixxx controls the output and recent Soundtouch compensates the input correct only at unity.